src/test/java/io/mosip/mimoto/service/IssuersServiceTest.java

```diff
@@ -41,9 +48,15 @@ public class IssuersServiceTest {
     @InjectMocks
     CredentialServiceImpl credentialService = new CredentialServiceImpl();
 
+    @Mock
+    RestApiClient restApiClient=new RestApiClient();
```

Comment:
What is this?
RestApiClient should be Autowired in IssuerServiceImpl and then here it can be only mock
```
@Mock
RestApiClient restApiClient;
```